src/marks/text.js

```diff
@@ -100,7 +103,7 @@ export class Text extends Mark {
           .enter()
           .append("text")
           .call(applyDirectStyles, this)
-          .call(applyMultilineText, this, T)
+          .call(applyMultilineText, this, T, channels.title)
```

Comment:
Maybe pass `channels` here directly and have applyMultilineText extract both _text_ and _title_ itself? Alternatively, you could pull out `channels.title` above where we destructure the other channels (though we’ll need a different name since `T` is already taken…). (You could probably pass `channels` to applyIndirectTextStyles, too, if you want to keep them consistent.)